src/App/actions/auth.js

```diff
@@ -12,86 +10,52 @@ import {
 
 import setAuthToken from "../utils/setAuthToken";
 
-const BaseURL = "https://osd-sidekick.herokuapp.com/api/auth";
-
-// const LocalURL = "http://localhost:8080/api/auth";
-
-// Load User
-
-export const loadUser = () => async dispatch => {
+export const loadUser = currentUser => async dispatch => {
   if (localStorage.token) {
     setAuthToken(localStorage.token);
   } else if (sessionStorage.token) {
     setAuthToken(sessionStorage.token);
   }
-
   try {
-    const res = await axios.get(`${BaseURL}`);
-
-    dispatch({
-      type: USER_LOADED,
-      payload: res.data
-    });
+    if (currentUser) {
+      dispatch({
+        type: USER_LOADED,
+        payload: currentUser
+      });
+    }
   } catch (err) {
     dispatch({
       type: AUTH_ERROR
     });
   }
 };
-// Register User
-export const register = ({ username, email, password }) => async dispatch => {
-  const config = {
-    headers: {
-      "Content-Type": "application/json"
-    }
-  };
-
-  const body = JSON.stringify({ username, email, password });
 
+// Register User
+export const register = registerData => async dispatch => {
```

Comment:
why do we still have the async without await?

Reply:
I forgot to remove `async` keyword. I'll change that in a second, thanks!